src/agent/onefuzz-supervisor/src/setup.rs

```diff
@@ -75,31 +78,33 @@ impl SetupRunner {
             let output = setup_script.invoke().await?;
 
             info!(
-                "setup script run; status = {}, path = {}",
-                output.status,
+                "setup script run; exit_status = {:?}, path = {}",
+                output.exit_status,
                 setup_script.path().display(),
             );
 
-            if output.status.success() {
+            if output.exit_status.success {
                 verbose!(
                     "setup script succeeded:\nstdout = {}\nstderr = {}",
-                    String::from_utf8_lossy(&output.stdout),
-                    String::from_utf8_lossy(&output.stderr),
+                    &output.stdout,
+                    &output.stderr,
                 );
                 info!("setup script succeeded");
             } else {
                 error!(
                     "setup script failed:\nstdout = {}\nstderr = {}",
-                    String::from_utf8_lossy(&output.stdout),
-                    String::from_utf8_lossy(&output.stderr),
+                    &output.stdout, &output.stderr,
                 );
-                anyhow::bail!("setup script failed:\nstdout = {}\nstderr = {}");
```

Comment:
This without the upcoming changes results in us silently ignoring a script failure. Rather than impl-ing a `bail!` where this is used, I'm just going to aim the next PR at this one for easy holistic review.

Reply:
Just reverted for now, so we can freely merge this.